config/openiotsdk/cmake/sdk.cmake

```diff
@@ -142,3 +169,50 @@ endif()
 # Add Open IoT SDK storage source
 add_subdirectory(${OPEN_IOT_SDK_STORAGE_SOURCE} ./sdk_storage_build)
 list(APPEND SDK_SOURCES_BINARY_DIRS ${CMAKE_CURRENT_BINARY_DIR}/sdk_storage_build)
+
+include(ElfToBin)
+include(TfmSignImage)
+
+function(sdk_post_build target)
+    if(TFM_SUPPORT)
+        string(REPLACE "_ns" "" APP_NAME ${APP_TARGET})
+        target_elf_to_bin(${APP_TARGET})
+        iotsdk_tf_m_sign_image(${APP_TARGET})
+        iotsdk_tf_m_merge_images(${APP_TARGET} 0x10000000 0x38000000 0x28060000)
```

Comment:
Why do we need to merge images ?

Reply:
It's more useful in the build script and Vscode tasks to have a single executable file. Then no more additional change is required and we don't need to have additional conditions for unit-test apps that don't need TF-M.